### DISCUSSION TITLE
Problema en el login

### DISCUSSION THREAD

**Discussion by jhondbv** [rramosp/ai4eng.v1#213] [01 - INTRODUCCIÓN]
Ingrese por medio de email y password (no login de google ) y cuando ejecuto el segundo comando del lab Practice submmision , tengo este error 
![image](https://user-images.githubusercontent.com/12846305/217314261-60fcb208-dc5d-4438-9aef-0dbd4f6972e7.png)

Intente cambiar los parametros de logueo para que tomara el curso 2023.01 , pero sigue sin poder loguear 
Tambien cambie la contraseña 
estoy intentando loguearme con email y password

**Comment by rramosp** [comment]
hola Jhon, el problema es que estabas en ambas sesiones. Ya lo arreglamos y
estás solo en la de 2023.1 ... prueba a entrar y confírmanos porfa si todo
está ok.

On Tue, Feb 7, 2023 at 12:07 PM jhondbv ***@***.***> wrote:

> Ingrese por medio de email y password (no login de google ) y cuando
> ejecuto el segundo comando del lab Practice submmision , tengo este error
> [image: image]
> <https://user-images.githubusercontent.com/12846305/217314261-60fcb208-dc5d-4438-9aef-0dbd4f6972e7.png>
>
> Intente cambiar los parametros de logueo para que tomara el curso 2023.01
> , pero sigue sin poder loguear
> Tambien cambie la contraseña
> estoy intentando loguearme con email y password
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/213>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIIYPKUYQLAL72I6GALWWJ6N7ANCNFSM6AAAAAAUUHGG7Y>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
